atom/browser/ui/win/notify_icon.cc

```diff
@@ -49,7 +49,18 @@ void NotifyIcon::HandleClickEvent(const gfx::Point& cursor_pos,
                                   bool left_mouse_click) {
   // Pass to the observer if appropriate.
   if (left_mouse_click) {
-    NotifyClicked();
+    NOTIFYICONIDENTIFIER icon_id;
+    memset(&icon_id, 0, sizeof(NOTIFYICONIDENTIFIER));
+    icon_id.uID = icon_id_;
+    icon_id.hWnd = window_;
+    icon_id.cbSize = sizeof(NOTIFYICONIDENTIFIER);
+
+    RECT rect;
```

Comment:
We should initialize it with zeros, otherwise when `Shell_NotifyIconGetRect` failed it would be filled with random numbers:

```c++
RECT rect = { 0 };
```